### PR TITLE
r-scPred recipe

### PR DESCRIPTION
**Pull request by a-solovyev12**
Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it. Note that if you are not already a member of the bioconda project (meaning that you can't add this label), please ping `@bioconda/core` so that your PR can be reviewed and merged (please note if you'd like to be added to the bioconda project). Please see https://github.com/bioconda/bioconda-recipes/issues/15332 for more details.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by a-solovyev12** [comment]
@BiocondaBot update

**Comment by a-solovyev12** [comment]
please review the recipe @bioconda/core. Thanks

File: recipes/r-scpred/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    - r-cowplot
+
+test:
+  commands:
+    - $R -e "library(scPred)"
+
+about:
+  home: https://github.com/IMB-Computational-Genomics-Lab/scPred
+  dev_url: https://github.com/IMB-Computational-Genomics-Lab/scPred/tree/development
+  license: unknown
```
Comment (dpryan79) [inline recipes/r-scpred/meta.yaml]: The license is actually GPL-3 for this package, can you update that here?

**Comment by dpryan79** [comment]
Thanks @a-solovyev12, can you repost your `please review the recipe @bioconda/core. Thanks` message? I wrote a github action just now that should repost things like that for non-members (to actually ping us) and this would let me actually test if it works :)

**Comment by a-solovyev12** [comment]
please review the recipe @bioconda/core. Thanks

**Comment by dpryan79** [comment]
Reposting to enable pings (courtesy of the BiocondaBot):
please review the recipe @bioconda/core. Thanks

**Comment by dpryan79** [comment]
Thanks, it seems the bot works (though I need to fix the formatting). I'll merge this once the CI finishes.

**Comment by a-solovyev12** [comment]
@dpryan79 great, thank you. Could you also add me to the bioconda project?

**Comment by dpryan79** [comment]
Yes, I'll do that when I merge this.

**Pull request merged**